tests/page/page-network-sizes.spec.ts

```diff
@@ -88,7 +89,10 @@ it('should have the correct responseBodySize for chunked request', async ({ page
   });
   const response = await page.goto(server.PREFIX + '/chunked-simplezip.json');
   const sizes = await response.request().sizes();
-  expect(sizes.responseBodySize).toBe(fs.statSync(asset('simplezip.json')).size);
+  if (browserName === 'webkit')
+    expect(sizes.responseBodySize).toBe(5173);
```

Comment:
Let's add a comment that it should be 5175, where the extra 75 are coming from. Also that webkit is not telling the truth here.